HISTORY.md

```diff
@@ -8,6 +8,7 @@
 * The EncryptionProvider::AddCipher method has been added to allow keys to be added to an EncryptionProvider.  This API will allow future providers to support multiple cipher keys.
 * Add a new option "allow_data_in_errors". When this new option is set by users, it allows users to opt-in to get error messages containing corrupted keys/values. Corrupt keys, values will be logged in the messages, logs, status etc. that will help users with the useful information regarding affected data. By default value of this option is set false to prevent users data to be exposed in the messages so currently, data will be redacted from logs, messages, status by default.
 * AdvancedColumnFamilyOptions::force_consistency_checks is now true by default, for more proactive DB corruption detection at virtually no cost (estimated two extra CPU cycles per million on a major production workload). Corruptions reported by these checks now mention "force_consistency_checks" in case a false positive corruption report is suspected and the option needs to be disabled (unlikely). Since existing column families have a saved setting for force_consistency_checks, only new column families will pick up the new default.
+* A new option, db_host_id, has been added to DBOptions. The option defaults to the hostname (FQDN). if set to something else,  the user specified string is written to SST files as a table property. If its empty, the property is not written.
```

Comment:
Myself coming from a case of people not fully digesting release notes and their potential impacts...

I suggest starting with the fact that SST files now include hostnames, but an option is added to override/exclude that.